CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 4.8.0 (2020-11-05)
+
+- Added country field testing fixture. [#145](https://github.com/blackbaud/skyux-lookup/pull/145)
+- Added input box to lookup demos. [#144](https://github.com/blackbaud/skyux-lookup/pull/144)
```

Comment:
You don't need to include any information on the demo/docs - that can stay internal.